packages/manager/modules/telecom-spare/src/modems/order/modems-order.routing.js

```diff
@@ -0,0 +1,29 @@
+import '@ovh-ux/manager-telecom-styles';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('spare.modems.order', {
+    url: '/order',
+    views: {
+      'spareView@spare': 'modemOrderComponent',
+    },
+    resolve: {
+      brands: /* @ngInject */ OvhApiXdsl => OvhApiXdsl.Spare()
+        .v6()
+        .getBrands()
+        .$promise.then((brands) => {
+          const spareBrands = [];
+          brands.sort().forEach((element) => {
+            spareBrands.push({
```

Comment:
`forEach` + `push`= `map`
So you can directly `return brands.sort().map(element => ({ etc … })`